c++11/src/tracer.cpp

```diff
@@ -1,8 +1,183 @@
+#include <collector.pb.h>
 #include <lightstep/tracer.h>
 #include <opentracing/noop.h>
+#include <memory>
+#include <cstdint>
+#include <mutex>
+#include <vector>
+using namespace opentracing;
```

Comment:
Will do. Do you think an `namespace ot = opentracing` alias would be a good idea to keep the names a little shorter?